### PR TITLE
Docs: Update sentence structure in user guide

### PR DESCRIPTION
**Pull request by taranjeet**
##### SUMMARY

This commit fixes a grammatically incorrect sentence in 
Introduction to Playbook Section under User Guide.

##### ISSUE TYPE

 - Docs Pull Request

##### COMPONENT NAME

User Guide > Introduction to Playbooks

### CODE REVIEW

File: docs/docsite/rst/user_guide/playbooks_intro.rst [hunk 1]
```diff
@@ -160,7 +160,7 @@ Support for running things as another user is also available (see :doc:`become`)
       remote_user: yourname
       become: yes
 
-You can also use become on a particular task instead of the whole play::
```
Comment (Akasurde) [inline docs/docsite/rst/user_guide/playbooks_intro.rst:163]: You can rephrase this as -

```
You can also use keyword 'become' on a particular task instead of the whole play

```

**Review by Akasurde — CHANGES_REQUESTED** [review]

File: docs/docsite/rst/user_guide/playbooks_intro.rst (hunk 1 above)
Comment (taranjeet) [reply docs/docsite/rst/user_guide/playbooks_intro.rst:163]: Yeah that suits better.

Comment (taranjeet) [reply docs/docsite/rst/user_guide/playbooks_intro.rst:163]: I have updated the sentence now. Thanks @Akasurde for the suggestion

**Review by acozine — CHANGES_REQUESTED** [review]
Hi @taranjeet, thank you for reading the documentation so carefully and for submitting a PR. We're committed to making the Ansible documentation world-class, and user feedback is an important part of that effort.

I think this change would be stronger if the word `become` were formatted as code - in rST, you do that with double back-ticks before and after the word.

For future documentation suggestions, please include a sentence or two in the PR letting us know what you found confusing or incorrect about the original wording. This will help us review and revise the user documentation.

**Comment by taranjeet** [comment]
@acozine : I have updated the `become` keyword. Also, in the future, I will take care that my commit messages are more meaningful.

**Review by Akasurde — APPROVED** [review]

**Review by bcoca — APPROVED** [review]

**Comment by Akasurde** [comment]
@taranjeet Thanks for the contribution. @bcoca @acozine Thanks for the review.
